internal/ota-api/dto.go

```diff
@@ -154,15 +159,68 @@ func (r OtaStatusDetail) String() string {
 	if len(r.Details) > 0 {
 		t = table.New()
 		t.SetHeader("Time", "Status", "Detail")
+		fwSize := int64(0)
+		if r.FirmwareSize != nil {
```

Comment:
I think you can also remove the pointer from `FirmwareSize`. IMHO there isn't a real reason to use a pointer

Reply:
I added a pointer because is doc it is stated that firmware size may not be present. In case it is missing, is it rmeoved from api response or returned as a 0 value?

Reply:
It shouldn't be present in the response, but anyway, for Golang that doesn't make any difference you will get a 0 either if the field is not present or if it is an actual 0.